tests/performance/tensor_eval/tensor_instructions.rb

```diff
@@ -0,0 +1,87 @@
+# Copyright Verizon Media. Licensed under the terms of the Apache 2.0 license. See LICENSE in the project root.
+
+require 'performance_test'
+require 'app_generator/search_app'
+require 'environment'
+
+class TensorInstructionBenchmarkTest < PerformanceTest
+
+  COST = 'cost'
+  IMPL_LONG = 'implementation'
+  IMPL_SHORT = 'impl_short'
+  BM_TYPE = 'benchmark_type'
+
+  def setup
+    super
+    set_owner("arnej")
+  end
+
+  def test_instruction_benchmark
+    set_description("Test performance of low-level tensor instructions")
+    deploy_app(SearchApp.new.sd(selfdir+"foobar.sd"))
```

Comment:
Consider renaming to `test.sd` and add the file to the PR.

Reply:
there's already a "test.sd" there, but havardpe suggested "dummy.sd" which makes it clear it's just a placeholder.

Reply:
👍